aioesphomeapi/_frame_helper.py

```diff
@@ -489,18 +489,8 @@ def _handle_frame(self, frame: bytearray) -> None:
                 ProtocolAPIError(f"Bad encryption frame: {ex!r}")
             )
             return
-        msg_len = len(msg)
-        if msg_len < 4:
-            self._handle_error_and_close(ProtocolAPIError(f"Bad packet frame: {msg!r}"))
-            return
```

Comment:
This should be impossible since the encryption would fail to decrypt if the packet was invalid